mne/viz/evoked.py

```diff
@@ -185,7 +185,7 @@ def _plot_evoked(evoked, picks, exclude, unit, show,
     titles = _handle_default('titles', titles)
     units = _handle_default('units', units)
     # Valid data types ordered for consistency
-    channel_types = ['eeg', 'grad', 'mag', 'seeg', 'dipole', 'gof']
+    channel_types = ['eeg', 'grad', 'mag', 'eog', 'seeg', 'dipole', 'gof']
```

Comment:
it also determines what can be plotted **at all**... let's see if I can separate the two.